docker/Dockerfile.alpine

```diff
@@ -0,0 +1,47 @@
+FROM nginxinc/nginx-unprivileged:1.20.2-alpine
```

Comment:
Can we pin the digest like we're doing it for the API server? https://github.com/DependencyTrack/dependency-track/blob/master/src/main/docker/Dockerfile#L1

Reply:
ofc! just dint bother yet as Renovate would have pined it for me xD (bad excuse for me being to lacy to grab the digest hashes, but will do now)